theme/scripts/index.ts

```diff
@@ -1 +1,17 @@
 import "../styles/index.scss";
+const navOpenMenuButton = document.getElementById('nav-open-menu-button');
+const navCloseMenuButton = document.getElementById('nav-close-menu-button');
+const navMobileMenu = document.getElementById('nav-mobile-menu');
+
+const openMenu = () => {
```

Comment:
Up to you, but might be nice to add remove a custom class that is a bit more descriptive, rather than adding removing multiple utils.

So you might have:

```
.MobileNav {
// Default styles

 &--active {
 // Active styles
 }
}
```

so now you should just be able to add remove `.MobileNav--active` (assuming you have the MobileNav base class on the MobileNav element by default)

Reply:
Great way to simplify my code, thank you!